package.json

```diff
@@ -13,6 +13,8 @@
     "build:react": "lerna run --scope hds-react build",
     "build:site": "lerna run --scope site build",
     "clean": "lerna clean --yes && rimraf node_modules/ packages/core/lib packages/core/storybook-static packages/react/lib packages/react/storybook-static packages/design-tokens/lib site/.cache site/public",
+    "prerelease": "yarn run lerna publish from-package --yes —dist-tag $npm_config_stage",
```

Comment:
Dear reviewers,

Whaddya say if we remove this `prerelease` script and call the `release` script straightaway from the workflow but with `-- --yes --dist-tag ${{ github.event.inputs.prerelease_stage }}` at the end? Because anything after the first `--` is passed to the script as is.

Reply:
Sounds better to use the `release` script and pass more props to it via `prerelase`